components/Wallet/Table/OpenOrdersTable.jsx

```diff
@@ -70,12 +71,25 @@ const OrderCancelButton = styled.button`
 export function OpenOrdersTable({ orders: _orders }) {
   // console.log(`OpenOrdersTable(`, arguments[0], `)`)
   const { t } = useTranslation('orders')
-  const [openOrdersData, setOpenOrdersData] = useState(_orders)
+
+  const _formattedOrders = useMemo(() => {
```

Comment:
Can you combine _orders and _formattedOrders into a single useState or useReducer? I think it might be incorrect also since the _orders are set after the _formattedOrders so the state isn't getting updated right away for the formatted orders.